src/utils/ioFileMod.F90

```diff
@@ -91,23 +91,23 @@ subroutine getfil(fulpath, locfn, iflag, lexist)
    if (len_trim(locfn) == 0) then
       call endrun ('(GETFIL): local filename has zero length')
    else if (masterproc) then
-      write(iulog,*)'(GETFIL): attempting to find local file ', trim(locfn)
+      write(iulog,'(a)')'(GETFIL): attempting to find local file '//trim(locfn)
    end if
- 
+
    inquire(file=locfn, exist=lexist_in)
    if (present(lexist)) lexist = lexist_in
    if (lexist_in) then
-      if (masterproc) write(iulog,*) '(GETFIL): using ',trim(locfn), ' in current working directory'
+      if (masterproc) write(iulog,'(a)') '(GETFIL): using '//trim(locfn)//' in current working directory'
```

Comment:
This would be more efficient as:
```suggestion
      if (masterproc) write(iulog,'(3a)') '(GETFIL): using ', trim(locfn), ' in current working directory'
```